programme/random-even-number/random-even-number.java

```diff
@@ -0,0 +1,24 @@
+//Program to generate random even number in range using Java
```

Comment:
```suggestion
//Programme to generate random even number in range using Java
```

For consistency we are now using `programme` everywhere and not `program`